prime-router/src/main/kotlin/fhirengine/engine/FHIREngine.kt

```diff
@@ -129,4 +130,16 @@ abstract class FHIREngine(
         actionLogger: ActionLogger,
         actionHistory: ActionHistory
     )
+
+    /**
+     * Given a [message], returns the topic that is embedded in the message. Defaults to full-elr since that was the
+     * first topic supported in the universal pipeline.
+     */
+    fun getTopicFromMessage(message: RawSubmission): Topic {
+        return Topic.fromJsonValue(message.topic) ?: let {
+            // This shouldn't happen, but if it does, log the error and reset to full-elr
+            logger.warn("Invalid topic ${message.topic} found in message, defaulting to 'full-elr'")
```

Comment:
This ties into my comment on deserializing into `Topic` rather having to go from the string, but I would we would want to blow up here?

Reply:
Updated RawSubmission to use `Topic` directly, so was able to remove this function.